src/firefly/js/visualize/ui/PngViewer.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {useStoreConnector} from '../../ui/SimpleComponent';
+import {getActiveTableId, getTblInfoById, getCellValue, getMetaEntry} from '../../tables/TableUtil';
+
+export function PngViewer() {
+    const [tbl_id] = useStoreConnector(() => () => getActiveTableId());
+    const {tableModel} = getTblInfoById(tbl_id);
+    if (!tableModel){
+        return null;
+    }
+    if(tableModel.isFetching){
+        return;
+    }
+    const highlightedRow = tableModel? tableModel.highlightedRow: 0;
+    const PngSource= getMetaEntry(tbl_id, 'ImagePreview', '');
```

Comment:
should be referred to by the constant
```
const PngSource= getMetaEntry(tbl_id, MetaConst.ImagePreview, '');
```

Reply:
variable name should not be uppercase unless the are classes, constants, or react component names
`pngSource`

Reply:
oops, that was a mistake! I will fix it.